modules/apps/dispatch/dispatch-test/src/testFunctional/tests/SwitchToTalendTabInJobScheduler.testcase

```diff
@@ -0,0 +1,49 @@
+@component-name = "portal-dispatch"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Job Scheduler";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@priority = 4
+	test CannotSeeErrorInConsole {
+		property portal.acceptance = "true";
+
+		task ("Given go to Control Panel > Job Scheduler") {
+			JobScheduler.openJobSchedulerAdmin();
+		}
+
+		task ("And Given edit one of the existing Job Scheduler etl-salesforce-product-connector-0.4.zip") {
+			JobScheduler.editJobItem(jobItemName = "etl-salesforce-product-connector-0.4.zip");
```

Comment:
This job only present when we ant all a dxp bundle, this will not be able to run on non EE environments, I will ask in slack channel to see if we can update the test to a more stable one to test the talend job